### ISSUE TITLE
Display the (linked) board filename next to the board number, in the sidebar

### ISSUE THREAD

**Issue by gordielachance** [wonderunit/storyboarder#1242]
**Is your feature request related to a problem? Please describe.**

I always have to open my board in Photoshop to see its filename.

**Describe the solution you'd like**
Would be better if we were able to display the filename next to the board "name".
This way we could also know if we are on a linked board or not.

![capture](https://user-images.githubusercontent.com/4077261/41719967-e1fcfbd2-7561-11e8-9751-bfc30f129593.JPG)

Even better, clicking the file name would open the explorer/finder at the place the file is (like "locate in explorer" for After Effects).

**Comment by audionerd** [comment]
I agree that we need a better way to indicate when a board is linked to a PSD. 

> which is not very intuitive where it is
Is that because "Show in Folder" is at the bottom of the sidebar so you have to scroll down every time to see it? Would it help if we had a keyboard shortcut for "Show in Folder"?

**Comment by gordielachance** [comment]
Yes, it is at the very bottom and anyway, it would be useful to see if there is a file linked at the first glance.
Thanks !

**Comment by gordielachance** [comment]
WOW ! That electron thing seems quite cool !!!
I did try to make my first code edit to fix this issue.

And...It... WORKS ! 

But I still need to update the current board metadata when creating a PSD / breaking the link.  Could you help me here ? First time I use Electron, any advice is welcome.

https://github.com/gordielachance/storyboarder/commit/d1f463c20689ed563fbcc790e0ebf1d6c57b599d

does board objects only have properties or do they have methods too ?

**Comment by audionerd** [comment]
Boards are just objects, not instances of a class.

When linking, the code in `openInEditor` runs.
When unlinking, the event handler for `requestUnlock` runs.
You could make sure that whenever `board.link` changes in those places, `renderMetaData` is called, and put your view code in there (that's the sidebar).

BTW I'm not yet convinced that the PSD name belongs in that spot in the sidebar for most users. But you're welcome to modify your own local copy of the app for now as we consider a better design.

**Comment by gordielachance** [comment]
Yeah. I'm just testing.

If you want to keep your design clean (which I understand); 
maybe an idea would be to have an icon to show that there is a file linked, but to display the filename in the tooltip when hovering the icon.
This way your design is kept gorgeous, but the information still is accessible for the users who needs it.

BTW what I would suggest is a template like this in the sidebar :

`Board: 1/3 [Edit externally icon] [Show In Folder icon]` <--icons are aligned right

This way, you can remove the Edit externally icon on the top bar (and gain space), while having all informations related to the board files in the same place (and thus have a more precise layout).
